client/src/index.js

```diff
@@ -5,36 +5,28 @@ import Home from "./components/Home";
 import "semantic-ui-css/semantic.min.css";
 import About from "./components/About";
 import Status from "./components/Status";
-import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { NavLink, BrowserRouter as Router, Route } from "react-router-dom";
+import { Menu, Container, Image } from "semantic-ui-react";
 import Footer from "./components/Footer";
+import banner from "./assets/banner.jpg";
 
 const Routes = () => {
   return (
     <Router>
+      <Menu inverted>
+        <Container>
+          <Menu.Item>
+            <Image size="small" src={banner} alt="Maryhill Integration logo" />
+          </Menu.Item>
+          <Menu.Item as={NavLink} to="/" name="home" />
```

Comment:
```suggestion
          <Menu.Item as={NavLink} exact to="/" name="home" />
```

This needs to be an exact match otherwise the home menu link is always shown as active.